extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -764,24 +764,6 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
        */
       get() = StyleManager.getStyleSourcePropertyDefaultValue("geojson", "generateId").silentUnwrap()
 
-    /**
-     * A property to use as a feature id (for feature state). Either a property name, or
-     * an object of the form `{<sourceLayer>: <propertyName>}`.
-     */
-    val defaultPromoteId: PromoteId?
```

Comment:
not sure we should remove the default getter, could you double check the line-gradient default property getter? we have similar situation there.

Reply:
default getters are used only for tests, so if tests are not present. we should remove it as well.

Reply:
These are public APIs 😄  So user could still decide to use them, if they are not used in our example app.

Reply:
Sorry, just checked line-gradient default property getter is also removed from the code, we are consistent.